handsontable/src/editors/autocompleteEditor/autocompleteEditor.js

```diff
@@ -127,11 +183,47 @@ export class AutocompleteEditor extends HandsontableEditor {
           }
         }
 
+        if (rootInstanceAriaTagsEnabled) {
+          setAttribute(TD, [
+            A11Y_OPTION(),
+            // Add `setsize` and `posinset` only if the source is an array.
+            ...(sourceArray ? [A11Y_SETSIZE(sourceSize)] : []),
+            ...(sourceArray ? [A11Y_POSINSET(sourceArray.indexOf(value) + 1)] : []),
+            ['id', `${this.htEditor.rootElement.id}_${value}`],
+          ]);
+        }
+
         TD.innerHTML = cellValue;
       },
-      autoColumnSize: true,
+      afterSelection: (startRow, startCol) => {
+        if (rootInstanceAriaTagsEnabled) {
+          const TD = this.htEditor.getCell(startRow, startCol, true);
+
+          if (TD) {
+            setAttribute(TD, [
+              A11Y_SELECTED(),
+            ]);
+          }
```

Comment:
```suggestion
          setAttribute(TD, [
            A11Y_SELECTED(),
          ]);
```
I guess the TD element is always there.